qiskit/converters/circuit_to_gate.py

```diff
@@ -12,6 +12,8 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
+# pylint: disable=no-member
+
```

Comment:
I think this disable belong to the `diagonal.py` class, as we had to do it for the other library objects. It might be a pylint artifact that it is required here.